### DISCUSSION TITLE
Journey to Quest NFT´s story continues

### DISCUSSION THREAD

**Discussion by LorDDark6660** [stellar/sorobanathon#90] [General]
### **Journey to Quest NFT´s** _story continues_

![soroban island](https://user-images.githubusercontent.com/92062140/204404254-30f4507e-b730-4c01-b575-fc10b34489cc.png)
Story Quest 5

https://litemint.com/items/GDFZ2RTA4THXLTAFNYQNJUBXXZROKHODXJ4OJ5TOCGQFLTK5IGNSEJHG/SQLJourney5

![Lucky fish](https://user-images.githubusercontent.com/92062140/205389712-08b55610-1376-41b5-8174-5e7aa57a6faa.png)
Story Quest 6

https://litemint.com/items/GAOKWXHKLUKN6NDGKFF64R4VP4VBVY6QQFGH5PXUQM47MCX4GFC7KW2Z/SQLJourney5

_To be continued..._

_Maybe like this?_

![to be continued](https://user-images.githubusercontent.com/92062140/205390933-2bc3cb1a-42ab-4f74-9ad1-04a9f142f20a.png)
To be continued

https://litemint.com/items/GAGCQRVPZM2B7PXAZK3ODBNNY2OLHTMWOFSWTI6NBIUB7RW3NOL6ZUH7/SQLJourney6

**_It was a great story, I didn't expect such an ending, excellent work !!!_**

Previous NFT´s here:
https://github.com/stellar/sorobanathon/discussions/43#discussion-4569064

**Comment by stellarjohn** [comment]
Hi LorDDark6660,

Congratulations!  Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 3900 XLM!  One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
